README.md

```diff
@@ -1,2 +1,55 @@
+  forces as well, which enormously increases the information content of the dataset.
+- **Include a variety of other information**.  SPICE includes a variety of other QM results for each sample,
+  such as bond orders, partial charges, and atomic multipoles.  It follows the principle that any result that
+  is easy to compute and potentially useful should be made available.
+- **Use an accuate level of theory**.  Models can never be more accurate than the data they are trained on.
+  SPICE computations are done at the ωB97M-D3BJ/def2-TZVPPD level of theory.
+- **Be a dynamic, growing dataset**.  The dataset should grow with time as new data is generated.  This will
+  allow models trained on it both to improve in accuracy and to expand the range of chemical space they cover.
+  Versioned releases will be created periodically to allow for reproducibility.
+- **Be freely available under a non-restrictive licence**.  All data in the SPICE dataset may be used under the
+  public domain equivalent [CC0 license](https://creativecommons.org/share-your-work/public-domain/cc0/).
+
+SPICE is made up of a collection of subsets.  Each one is designed to provide a particular type of information.
+They include the following.
+
+- **Dipeptides**.  These provide comprehensive sampling of the covalent interactions found in proteins.
+- **Solvated amino acids**.  These provide sampling of protein-water and water-water interactions.
+- **PubChem molecules**.  These sample a very wide variety of drug-like small molecules.
+- **Monomer and dimer structures from [DES370K](https://www.nature.com/articles/s41597-021-00833-x)**.
+  These provide sampling of a wide variety of non-covalent interactions.
+- **Ion pairs**.  These provide further sampling of Coulomb interactions over a range of distances.
+
+## Getting The Data
+
+This repository contains scripts and data files used in creating the dataset.  The SPICE dataset itself
+is hosted on [QCArchive](https://qcarchive.molssi.org/).  It can be obtained in a few ways.
+
+First, the [Releases page](https://github.com/openmm/spice-dataset/releases) provides the data for each
+release as a single HDF5 file.  Because some data types can be very large, these files include only the
+most commonly used results: total energies, formation energies, and forces.
+
+Second, the [downloader script](https://github.com/openmm/spice-dataset/tree/main/downloader) can be used
+to create a HDF5 file with whatever data fields you need.  This is useful when you want less commonly used
+fields, such as bond orders or atomic multipoles.  The page linked above describes the format of the HDF5
+files and has instructions on how to configure what information to download.
+
+Third, the data can be retrieved using the [QCPortal library](https://docs.qcarchive.molssi.org/projects/QCPortal/en/stable/).
+It provides a programmatic API for querying and accessing data.
+
+## Citing The Dataset
```

Comment:
If you enable the [Zenodo GitHub integration](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content), when you cut a release in GitHub, a DOI will automatically be assigned along with a citable reference. Even better, it will pull the list of contributors from a [CITATION.cff](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) file if we create one for the repo. This will allow each version of the dataset to have its own independent DOI and citation that we can use until a preprint is posted. Presumably, the preprint would be the primary citation after that, so that users could cite "the OpenMM SPICE QM dataset 1.0.3", but it doesn't hurt for each version of the filtered dataset to have its own DOI in Zenodo.

Reply:
Enabling Zenodo integration is a great idea.  I'll look into it.